poc/gen_test_vec.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+import os
+
+from vdaf_poc.test_utils import gen_test_vec_for_vdaf
+
+from vdaf_prio3_l1_bound_sum import Prio3L1BoundSum
+
+TEST_VECTOR_PATH = os.environ.get(
+    "TEST_VECTOR_PATH",
+    "../test_vec/",
+)
+
+
+if __name__ == "__main__":
+    ctx = b"some application"
+    vdaf_test_vec_path = TEST_VECTOR_PATH + "/vdaf/"
+
+    num_shares = 2
+    prio = Prio3L1BoundSum(2, 10, 8, 9)
```

Comment:
I'm not going to pretend that I understand all of these unlabeled parameters, but I will observe that they are unlabeled and that is not good.  The effect is that this is effectively write-only code, because you need a ton of context to understand any part of it.  That is a bit unfortunate; I'd encourage you to spend some time on improving that, though realize that this is a part of a much larger framework that would also need to move, so it's unlikely.

Reply:
Fair point, I've been leaning on my IDE for this. Python lets me specify positional arguments as keyword arguments, so I'll go through and do that in a follow-up.